src/data/set/accumulate.lean

```diff
@@ -0,0 +1,50 @@
+/-
+Copyright (c) 2020 Floris van Doorn. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Floris van Doorn
+-/
+import data.set.lattice
+import data.finset.basic
+/-! ### Accumulate
```

Comment:
Why do you use `###` instead of `#` here? And AFAIR `doc-gen` sometimes fails to parse `/-! ###` correctly.
```suggestion
/-!
### Accumulate
```

Reply:
Oops, to used to subheaders. Ok. I'll try to remember to put `#`'s on a new line from now on.